### PR TITLE
Improve exception message in customiseEarlyDelete

### PR DESCRIPTION
**Pull request by makortel**
#### PR description:

The failed tests in #26811 revealed that in case a module uses `refToPSet_` to point to a global `PSet` that is not found from the `Process`, the exception message comes from the `getattr()` and is not very helpful. This PR improves the exception message by including the offending module label and the referenced-to PSet label.

#### PR validation:

Unit tests pass.

### CODE REVIEW

**Comment by makortel** [comment]
FYI @Dr15Jones

**Comment by makortel** [comment]
@cmsbuild, please test

File: Configuration/StandardSequences/python/earlyDeleteSettings_cff.py [hunk 1]
```diff
@@ -122,11 +126,19 @@ def testHasInputTagModuleLabel(self):
...
 
             result=[False,False,False,False,False,False,False,False,False,False,False,False,False,False]
-            _hasInputTagModuleLabel(p, p.prod, ["foo","foo2","foo3","bar","fred","wilma","a","foo4","bar2","bar3","fred2","wilma2","a2","joe"],result)
+            _hasInputTagModuleLabel(p, p.prod, "prod", ["foo","foo2","foo3","bar","fred","wilma","a","foo4","bar2","bar3","fred2","wilma2","a2","joe"], result)
             for i in range (0,13):
                 self.assert_(result[i])
             self.assert_(not result[13])
 
+            result = [False]
+            self.assertRaises(RuntimeError, _hasInputTagModuleLabel, p, p.prod2, "prod2", ["foo"], result)
```
Comment (fabiocos) [inline Configuration/StandardSequences/python/earlyDeleteSettings_cff.py]: @makortel what would be the expected behaviour here? If I interpret correctly https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises I would expect an exception, running the unit test gives

```
15:04 cmsdev25 1679> python Configuration/StandardSequences/python/earlyDeleteSettings_cff.py
.
----------------------------------------------------------------------
Ran 1 test in 0.055s

OK
```

Comment (makortel) [reply Configuration/StandardSequences/python/earlyDeleteSettings_cff.py]: If the function raises a `RuntimeError' exception, the `assertRaises()` succeeds. If the function does not raise an exception, or it raises an exception of some other type, the `assertRaises()` fails.

**Comment by fabiocos** [comment]
+operations

the update appears coherent with the purpose of the PR

**Comment by fabiocos** [comment]
+1

**Pull request merged**